### ISSUE TITLE
Replicating results of AlphaTensor

### ISSUE THREAD

**Issue by afzalxo** [kurtosis/mat_mul#1]
Hi, I am trying to replicate the results of AlphaTensor for 4x4 matrices. I ran the training script with the default settings but the reward never goes below -7. May I ask if you replicated any of the results? If so, do you have the training hyperparameters that could allow reproducing the results of matrix multiplication problem of any size? Although Deepmind used a lot of compute, I feel we should be able to reproduce results for small matrices with the amount of compute available to a typical lab.

Thanks
